descriptor.md

```diff
@@ -61,22 +61,33 @@ The _digest_ property of a Descriptor acts as a content identifier, enabling [co
 It uniquely identifies content by taking a [collision-resistant hash](https://en.wikipedia.org/wiki/Cryptographic_hash_function) of the bytes.
 If the identifier can be communicated in a secure manner, one can retrieve the content from an insecure source, calculate the digest independently, and be certain that the correct content was obtained.
 
-The value of the digest property, the _digest string_, is a serialized hash result, consisting of an _algorithm_ portion and a _hex_ portion.
-The algorithm identifies the methodology used to calculate the digest; the hex portion is the lowercase hex-encoded result of the hash.
+The value of the digest property, the _digest string_, is a serialized hash result, consisting of an _algorithm_ portion and an _encoded_ portion.
+The algorithm identifies the methodology used to calculate the digest and serialize the _encoded_ portion.
 
 The digest string MUST match the following grammar:
 
 ```
-digest      := algorithm ":" hex
-algorithm   := /[a-z0-9_+.-]+/
-hex         := /[a-f0-9]+/
+digest                := algorithm ":" encoded
+algorithm             := algorithm-component [algorithm-separator algorithm-component]*
+algorithm-component   := /[a-z0-9]+/
+algorithm-separator   := /[+._-]/
+encoded               := /[a-zA-Z0-9=_-]+/
 ```
 
+Implementations SHOULD allow digests that are unsupported to pass validation if they comply with the above grammar.
+While `sha256` will only use hex encoded digests, support for separators in _algorithm_ and alpha numeric in _encoded_ is included to allow for future extension of digest support.
+As an example, we can parameterize the encoding and algorithm as `multihash+base58:QmRZxt2b1FVZPNqd8hsiykDL3TdBDeTSPX9Kv46HmX4Gx8`, which would be considered valid but unsupported by this specification.
+
 Some example digest strings include the following:
 
-digest                                                                  | algorithm           |
-------------------------------------------------------------------------|---------------------|
-sha256:6c3c624b58dbbcd3c0dd82b4c53f04194d1247c6eebdaab7c610cf7d66709b3b | [SHA-256](#sha-256) |
+digest                                                                     | algorithm           | Supported |
```

Comment:
“Supported” isn't particularly clear to me.  I think there are two interesting levels for algorithm identifiers in the spec:

* Registered (e.g. listed [here][1]).
* Required to be implemented (as we do for `sha256` [here][2]).

For example, `sha512` (in flight with #609) would be “registered” but would not be “required to be implemented”.

I'm in favor of staying DRY and dropping the “Supported” column here, but I'd also be ok with a “Registered” and/or “Required to be implemented” columns.  Without a clearer definition, I'm not in favor of a “Supported” column.

[1]: https://github.com/opencontainers/image-spec/blob/1eb142309125cdf562d7cca8c279836cd6c04140/descriptor.md#registered-identifiers
[2]: https://github.com/opencontainers/image-spec/blame/1eb142309125cdf562d7cca8c279836cd6c04140/descriptor.md#L123